google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -543,7 +543,7 @@ private Value() {}
    *
    * @throws IllegalStateException if {@code isNull()} or the value is not of the expected type
    */
-  abstract List<Struct> getStructArray();
+  public abstract List<Struct> getStructArray();
```

Comment:
@olavloite I made this method public. It surprised me this was not public before, since users can `SELECT ARRAY(SELECT STRUCT ...)` and would not be able to get the results.